doc/en/components/grids/grid/overview.md

```diff
@@ -56,28 +56,38 @@ Afterwards, you may start implementing the control by adding the following names
 ```razor
 @using IgniteUI.Blazor.Controls
 ```
-
 <!-- end: Blazor -->
 
 <!-- Angular, React, WebComponents -->
 When installing the {Platform} grid package, the core package must also be installed.
 
-<pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
+```cmd
 npm install --save {PackageCore}
 npm install --save {PackageGrids}
 npm install --save {PackageInputs}
```

Comment:
Actually, this seems to be somewhat outdated too, in the [package.json](https://unpkg.com/browse/igniteui-webcomponents-grids@4.0.3/package.json) for grids there are 3 peer dependencies - core, inputs and layouts and all should be installed. So add layouts too. 
PS: These can be bundled into a single install command btw